template.toml

```diff
@@ -41,8 +41,8 @@ diagnostic = { } # Code diagnostics in editing area (LSP)
 'ui.window' = { } # ?
 
 'ui.text' = { } # Default text color
-'ui.text.focus' = { } # ?
-'ui.text.info' = { } # ?
+'ui.text.focus' = { } # Selection highlight in buffer-picker or file-picker
+'ui.text.info' = { } # Info popup contents (space mode menue)
```

Comment:
Why the spelling of "menue" with an e?

Reply:
Typo, i fix it with the next pr if you like

Reply:
You can commit to your own branch and git push and it will automatically show up here, saves the bother of multiple PRs

Reply:
Hmm `git push origin adding-some-parts` looks like the right command. I don't see any PRs on your github, closed or opened. Don't stress about it, I know how it feels :+1:.

Reply:
thanks man :>
i try the push origin